test/bandit/http1/request_test.exs

```diff
@@ -329,6 +329,41 @@ defmodule HTTP1RequestTest do
       send_resp(conn, 200, "OK")
     end
 
+    test "reads a content-length with multiple content-lengths encoded body properly", context do
+      {:ok, response} =
+        Finch.build(
+          :post,
+          context[:base] <> "/expect_body_with_multiple_content_length",
+          [{"connection", "close"}, {"content-length", "8000000,8000000,8000000"}],
+          String.duplicate("a", 8_000_000)
+        )
+        |> Finch.request(context[:finch_name])
```

Comment:
Maybe add a test to cover the case where they're comma separated, but different from one another? The twists and turns of the RFC are pretty nasty and there's no better time to pile on the tests than now when we understand them.

Also maybe drop the relevant RFC section in a comment to remember why we're testing them :)

Reply:
Added a test and a comment to reference the RFC